Makefile

```diff
@@ -60,6 +60,11 @@ test/e2e-latest-image:
 	# Doesn't need cluster/prepare as it's done by operator-sdk. Uses a randomly generated namespace (instead of keycloak namespace) to support parallel test runs.
 	operator-sdk run local ./test/e2e --go-test-flags "-tags=integration -coverpkg ./... -coverprofile cover-e2e.coverprofile -covermode=count" --debug --verbose
 
+.PHONY: test/e2e-image-no-admin
```

Comment:
I would prefer to replace one of our existing targets with this. The `test/e2e-latest-image` seems to be a good candidate.

Reply:
It's basically a small hack to run the tests this way – we need to use custom manifests etc. I'm not sure we want to use it on regular basis (like `test/e2e-latest-image`). Also, you still need to manually run `make cluster/prepare` as an admin before this, so I don't really we should replace some existing target with this. This is really a special use case.

Reply:
Ok, understood. 

And where does it fail? What if we break it somewhere in the upstream? 

I would like to avoid a situation where we know about a potential failure late in the loop. If this configuration is needed, it should be tested in Travis in my opinion.